README.md

```diff
@@ -179,6 +179,14 @@ Cron, every, and interval types push jobs into sidekiq in a recurrent manner.
     every: '45m'    # Runs every 45 minutes
 ```
 
+The value is parsed by `Fugit::Duration#parse`. It understands quite a number of formats, including human-readable ones:
```

Comment:
```suggestion
The value is parsed by  [Fugit](https://github.com/floraison/fugit). It understands quite a number of formats, including human-readable ones:
```
What do you think about linking to the gem? I think that can be more useful to the other users

Reply:
Good idea! But linking _just_ to the gem isn't precise enough. How about this anchored link?

Reply:
Looks good to me!